### ISSUE TITLE
Error trying to revert cursor change properties

### ISSUE THREAD

**Issue by enriquepiatti** [amcharts/amcharts4#1528]
I want to change the cursor lineX during some time (like some type of x coordinate selector) and then revert it back to the original style.    
I'm doing something like:   
```
chartpools.cursor.xAxis = chartpools.xAxes.getIndex(0);
chartpools.cursor.fullWidthLineX = true;
```
And that works, the cursor is changed, but then I cannot revert it, this trigger an error:  
`chartpools.cursor.xAxis = undefined;`
The error is:

> charts.js:1 Uncaught TypeError: Cannot read property 'tooltip' of undefined
>     at e.set [as xAxis] (charts.js:1)
>     at <anonymous>:1:25

And this do nothing:
`chartpools.cursor.fullWidthLineX = false;`

**Comment by martynasma** [comment]
Must be a bug. Thanks!

Easy demo for testing:
https://codepen.io/team/amcharts/pen/059628c656ce7de5c78e0eac41bd39cd?editors=0010

**Comment by enriquepiatti** [comment]
currently the workaroung I'm using (in case is helpful for someone else) is to create a new cursor instead of reverting the properties.

**Comment by martynasma** [comment]
Thanks for sharing the workaround. That sounds a bit excessive, tho. We need to solve this, or find some other way.

**Comment by martynasma** [comment]
Fixed in [4.5.14](https://github.com/amcharts/amcharts4/releases/tag/4.5.14).

## [4.5.14] - 2019-08-11

### Added
- `mouseOptions` added to `Interactions`. For now it contains only wheel sensitivity setting, e.g. `mapChart.interactions.mouseOptions.sensitivity = 0.4`.
- `getPositionX()` and `getPositionY()` methods added to all axes. Returns relative position of a series data item on axis.

### Changed
- Series with `DateAxis` not as a base axis of a series (like gantt) will now animate similar to other charts.
- `SlicedChart` hidden labels arrangements adjusted.

### Fixed
- `ValueAxis` was incorrectly choosing min/max values for the data with very small values.
- `StepLineSeries` with `connect = false` and `autoGapCount > 1` was showing regular line series, not a step series.
- Chart with `DateAxis` which was not a base axis of a series and dates older than Unix Epoch (January 1st, 1970 at UTC) was incorrectly showing base of a series.
- `DateAxis` was not animating smoothly when adding more data.
- `ValueAxis` pre-zoom (on `ready` event) was not working properly.
- `fullWidth` cursor line was not reverting back if set to `true` and then back to `false`.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
